backend/docker.go

```diff
@@ -165,6 +168,15 @@ func newDockerProvider(cfg *config.ProviderConfig) (Provider, error) {
 		execCmd = strings.Split(cfg.Get("EXEC_CMD"), " ")
 	}
 
+	inspectInterval := defaultInspectInterval
+	if cfg.IsSet("INSPECT_INTERVAL") {
+		v, err := strconv.ParseInt(cfg.Get("INSPECT_INTERVAL"), 10, 64)
```

Comment:
I would prefer using `time.ParseDuration` instead, which means we don't have to assume milliseconds.